video/csputils.h

```diff
@@ -139,31 +141,14 @@ extern const struct m_opt_choice_alternatives mp_stereo3d_names[];
 #define MP_STEREO3D_NAME_DEF(x, def) \
     (MP_STEREO3D_NAME(x) ? MP_STEREO3D_NAME(x) : (def))
 
-struct mp_hdr_metadata {
-    // HDR10
-    // Mastering display metadata
-    float min_luma, max_luma;       // min/max luminance (in cd/m²)
-
-    // Content light level
-    float max_cll;                  // max content light level (in cd/m²)
-    float max_fall;                 // max frame average light level (in cd/m²)
-
-    // HDR10+
-    float scene_max[3];             // maxRGB in cd/m² per component (RGB)
-    float scene_avg;                // average of maxRGB in cd/m²
-
-    // CIE Y
-    float max_pq_y;                 // maximum PQ luminance (in PQ, 0-1)
-    float avg_pq_y;                 // averaged PQ luminance (in PQ, 0-1)
-};
-
 struct mp_colorspace {
     enum mp_csp space;
     enum mp_csp_levels levels;
     enum mp_csp_prim primaries;
     enum mp_csp_trc gamma;
     enum mp_csp_light light;
-    float sig_peak; // highest relative value in signal. 0 = unknown/auto
+    float sig_peak; // highest relative value in signal. 0 = unknown/auto (deprecated)
```

Comment:
do you plan to also work on ripping this out?
there isn't much sense to deprecating something that's internal to mpv otherwise

Reply:
Yes, eventually. Didn't plan doing it now, but can look into it. There is small difference how sig_peak is handled, because it was one value that was either maxCLL or maxLuminance, but apart from that it is replaceable.

It is exposed as `video-param/sig-peak`, but I guess we have `M_PROPERTY_DEPRECATED_ALIAS` and maybe it works for subprop too.

Reply:
and done, PR got little bit bigger.